tests/subsys/zbus/dyn_channel/testcase.yaml

```diff
@@ -1,5 +1,4 @@
 tests:
-  dyn_channel.static_and_dynamic_channels:
-    build_only: false
+  message_bus.dyn_channel.static_and_dynamic_channels:
```

Comment:
If the goal is to categorise Zbus tests, why not use `zbus` instead of `message_bus`?

Reply:
Maybe zbus make more sense for this case.

Reply:
The idea is to have the feature, not the name of the subsystem in zephyr, thats why we have `testing.ztest`
 and not `ztest`, etc., could be changed to `message_bus.zbus`, I am fine with that.

Reply:
> message_bus.zbus

Understood. It makes sense. `message_bus.zbus` is good.